app/pages/cif/projects.tsx

```diff
@@ -9,9 +9,22 @@ import { useRouter } from "next/router";
 import { getProjectRevisionPageRoute } from "pageRoutes";
 import Table from "components/Table";
 import ProjectTableRow from "components/Project/ProjectTableRow";
+import {
+  DisplayOnlyFilter,
+  NoHeaderFilter,
+  SortOnlyFilter,
+  TextFilter,
+} from "components/Table/Filters";
 
 export const ProjectsQuery = graphql`
-  query projectsQuery {
+  query projectsQuery(
+    $projectName: String
+    $operatorTradeName: String
+    $rfpNumber: String
+    $offset: Int
+    $pageSize: Int
+    $orderBy: [ProjectsOrderBy!]
+  ) {
```

Comment:
Will need to add the $status variable here to make the Project Status column searchable
```suggestion
  query projectsQuery(
    $projectName: String
    $operatorTradeName: String
    $rfpNumber: String
    $status: String
    $offset: Int
    $pageSize: Int
    $orderBy: [ProjectsOrderBy!]
  ) {
```